lutris/pga.py

```diff
@@ -90,6 +90,7 @@ def migrate_games():
         {'name': 'has_custom_banner', 'type': 'INTEGER'},
         {'name': 'has_custom_icon', 'type': 'INTEGER'},
         {'name': 'playtime', 'type': 'TEXT'},
+        {'name': 'playscript', 'type': 'TEXT'},
```

Comment:
wth is a new field needed in the PGA for that!?!??!!?!?!?

Reply:
This and the one after are just guessing, for the pga I thought when i add the play_script to it, when someone makes an installer he can create a script and point  play_script to it,

 for example a battle net installer, with play_script that deletes Application Data/battle.net folder which is a frequent workaround, the installer might say for example if u have problem starting battle net click play script than try again

Reply:
and for things like #681 basically the installer would say if u want to read the manual hit play_script, the script would have for example (xdg-open manual.pdf)